### PR TITLE
media/audio_manager : find all card when control volume if there's no…

### PR DESCRIPTION
**Pull request by Taejun-Kwon**
… actual card

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [1d7d1203743e28a8ea70c010caa7f17be596ad13] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [1d7d1203743e28a8ea70c010caa7f17be596ad13] - Code Rule Check OK.

File: framework/src/media/audio/audio_manager.c [hunk 1]
```diff
@@ -217,54 +217,64 @@ audio_manager_result_t find_audio_card(audio_card_type_t card_type)
 	return ret;
 }
 
+static audio_manager_result_t find_audio_card_if_empty(audio_card_type_t card_type)
```
Comment (jarvisjang) [inline framework/src/media/audio/audio_manager.c]: It looks that considering INPUT card_type is needed.

**Comment by seinfra** [comment]
Target : [28404b41e14a57d271b221f7189f3a8dc5fda761] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [28404b41e14a57d271b221f7189f3a8dc5fda761] - Code Rule Check OK.

File: framework/src/media/audio/audio_manager.c (hunk 1 above)
Comment (Taejun-Kwon) [reply framework/src/media/audio/audio_manager.c]: Fixed

File: framework/src/media/audio/audio_manager.c [hunk 2]
```diff
@@ -217,54 +217,64 @@ audio_manager_result_t find_audio_card(audio_card_type_t card_type)
...
+{
+	if ((g_actual_audio_out_card_id == INVALID_ID) && (find_audio_card(card_type) != AUDIO_MANAGER_SUCCESS)) {
+		meddbg("Found no active output audio card\n");
```
[the remark this answers is not in the document]
Comment (Taejun-Kwon) [reply framework/src/media/audio/audio_manager.c]: Thanks, fixed.

File: framework/src/media/audio/audio_manager.c [hunk 3]
```diff
@@ -217,54 +217,70 @@ audio_manager_result_t find_audio_card(audio_card_type_t card_type)
...
 
+static audio_manager_result_t find_audio_card_if_empty(audio_card_type_t card_type)
+{
+	int id = g_actual_audio_out_card_id;
+
+	if (card_type == INPUT) {
+		id = g_actual_audio_in_card_id;
+	}
+	
+	if ((id == INVALID_ID) && (find_audio_card(card_type) != AUDIO_MANAGER_SUCCESS)) {
```
Comment (jsdosa) [inline framework/src/media/audio/audio_manager.c]: @Taejun-Kwon is that correct condition~?
i think change to or codition

Comment (Taejun-Kwon) [reply framework/src/media/audio/audio_manager.c]: yes it is correct. if id is INVALID_ID, it means there's no founded card.
Hence find_audio_card called.

**Comment by jsdosa** [comment]
@Taejun-Kwon look so good.

**Pull request merged**